src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -412,22 +376,25 @@ def __init__(self, project_id, raw_response):
 
         for hit in raw_response['hits']['hits']:
             if hit['_id'] == project_id:
+                hit_contents = hit['_source']['contents'].to_dict()
```

Comment:
This loop bugs me. I wanted to eliminate it but failed to do so because of the aggregates being computed by ES. Instead, I added the assert below. With your fixes in place I think it can now be eliminated.